server/public/partials/statblock/statBlock.html

```diff
@@ -62,7 +62,7 @@
 
     <div class="creature-heading">
       <h1 style="margin-right:40px;">{{creature.name}}</h1>
-      <h2>{{creature.stats.size.charAt(0).toUpperCase() + creature.stats.size.slice(1).toLowerCase()}} {{creature.stats.race.toLowerCase()}}, {{creature.stats.alignment.toLowerCase()}}</h2>
+      <h2>{{creature.stats.size.charAt(0).toUpperCase() + creature.stats.size.slice(1).toLowerCase()}} {{creature.stats.race}}, {{creature.stats.alignment}}</h2>
```

Comment:
I'd suggest following the example of stats.size, where it explicitly capitalizes the first character and sets the rest to lowercase. It's more verbose, but it serves to decouple the presentation from the data. (sorry, didn't notice this on the first pass)

Reply:
Sure, makes sense to me